### DISCUSSION TITLE
为什么<Search />这个icon会自动带一个input出来 如何去掉

### DISCUSSION THREAD

**Discussion by 2010363329** [lucide-icons/lucide#1715] [Q&A]
help

**Comment by karsa-mistmere** [comment]
You're probably importing the wrong `Search`.

Easiest way to avoid this name collision is to use `<SearchIcon/>` or `<LucideSearch/>` instead.

**Reply by 2010363329** [reply]
Yes, I've corrected it. Thank you. I'm closing this discussion
